src/modules/Category/index.tsx

```diff
@@ -71,10 +71,26 @@ export const Category = () => {
     setOpen(true);
   };
 
+  const resetForm = () => {
```

Comment:
We have `formik` in our dependencies, any specific reason to not use form library and manage state ourselves?

Reply:
Yes we have it but for this form formik is not used.

Reply:
It was Previously written code , it is not resetting the form on cancel button so I added it.

Reply:
We should plan on migrating forms in the app to a form library when we are using one.